src/solvers/smt2/smt2_conv.cpp

```diff
@@ -42,7 +41,7 @@ Author: Daniel Kroening, kroening@kroening.com
 #define UNEXPECTEDCASE(S) throw "Unexpected case: " S
 
 // General todos
-#define TODO(S) throw "TODO: " S
+#define SMT2_TODO(S) throw "TODO: " S
```

Comment:
I wanted this commit to be as minimal as possible and to be clearly not changing functionality (the whole set should be very low risk / minimal impact).  To replace with the new TODO I'd have to move the more complex error messages to use error(), which would mean a bigger change.